helper/discover.js

```diff
@@ -1,13 +1,20 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 /* eslint-disable import/no-commonjs */
 
-const compareVersions = require('compare-versions');
+const { compare } = require('compare-versions');
 const exec = require('child_process').exec;
 
 const minimumVersion = '1.3.5';
 let browser;
 let nodes = [];
 
+function compareVersions(left, right, direction) {
+  function fixVersion(v) {
+    return v.replace('rc', '-rc').replace('--rc', '-rc');
```

Comment:
Have you tested with some of the other version tags OctoPrint will create if running out of a git branch? I don't think thats something that needs to be supported now but I'll probably want support for that eventually.

Reply:
I have not, but with my own experience and knowledge of how `git describe` works, it should work in theory!